internal/websocket/ws.go

```diff
@@ -266,6 +269,8 @@ func (ws *Websocket) run(ctx context.Context) {
 						Type: event.PING,
 					})
 				})
+
+				inactivityTimeout = time.After(ws.inactivityTimeout)
```

Comment:
based on how `typ, reader, err := ws.conn.Reader(ctx)` is implemented, there's no race condition from what I see (`go test --race` comes back clean as well)